pkg/controllers/vdb/onlineupgrade_reconciler.go

```diff
@@ -213,6 +220,8 @@ func (r *OnlineUpgradeReconciler) Reconcile(ctx context.Context, _ *ctrl.Request
 		r.postStartReplicationMsg,
 		r.startReplicationToReplicaGroupB,
 		r.waitForReplicateToReplicaGroupB,
+		// replicate v_internal_tables.v_redirect_state to sandbox
+		r.copyRedirectStateToReplicaGroupB,
```

Comment:
Can you remind me why we need this?

Reply:
eventually this state should be copied as part of replicate. until that point, the operator needs to copy over the key that is used for session redirection